common/src/main/webapp/less/style.less

```diff
@@ -1504,10 +1502,21 @@ html {
   border: 0 !important; // important to override SUI's style.
 }
 
-.explore-button-dismiss {
-  border: 0 !important; // important to override SUI's style.
-}
-
 .explore-button-summary {
   width: 100%;
 }
+
+.explore-tile-title-button-new {
+  margin-left: 1em !important; // important to override SUI's style.
+}
+
+// This fixes what seems to be a bug in SUI (React?), where compact button groups are still very wide.
```

Comment:
Is it mentioned as a bug somewhere? Perhaps in FomanticUI has been fixed?

Reply:
I dag a bit deeper and it's not actually a bug. The thing is, labeled+`icon` buttons should actually have a `labelPosition` or they are not rendered as expected (they are rendered much smaller, which is what I want in this case).

So, I'm fixing this to introduce a `verycompact` style that we can apply to buttons to render them as we want.